pkg/balancer/round_robin.go

```diff
@@ -15,13 +15,14 @@
 package balancer
 
 import (
-	"sync"
 	"sync/atomic"
+
+	"github.com/tikv/pd/pkg/utils/syncutil"
 )
 
 // RoundRobin is a balancer that selects nodes in a round-robin fashion.
 type RoundRobin[T uint32 | string] struct {
-	sync.RWMutex
+	syncutil.RWMutex
```

Comment:
What difference are there?

Reply:
If we run `make test`, it will use the deadlock RWMutex.